client/flutter/lib/pages/question_index.dart

```diff
@@ -149,7 +151,7 @@ class _QuestionTileState extends State<QuestionTile>
               child: Html(
                 data: widget.questionItem.title,
                 defaultTextStyle: _titleStyle.copyWith(
-                  fontSize: 16 * MediaQuery.of(context).textScaleFactor,
+                  fontSize: 18 * MediaQuery.of(context).textScaleFactor,
```

Comment:
I know this was from a prior change, but... shouldn't we remove textScaleFactor? I think this will apply the scale factor twice, no?

Reply:
Html widget does not account for textScaleFactor.